`make serve` broken on master

`make serve` does not currently work on master:

```
rbellevi@rbell:~/Dev/grpc.io$ make serve
hugo server \
        --buildDrafts \
        --buildFuture \
        --disableFastRender
Built in 192 ms
Error: Error building site: TOCSS: failed to transform "sass/style.sass" (text/x-sass): resource "sass/sass/style.sass_5538683f21764355e16f4f866a5e0357" not found in file cache
make: *** [Makefile:9: serve] Error 255
rbellevi@rbell:~/Dev/grpc.io$ hugo version
Hugo Static Site Generator v0.67.1-4F44227B linux/amd64 BuildDate: 2020-03-15T19:31:42Z
```

d721e7c2d028a5369d9ffe840ac6f65015638940 appears to be the first bad commit:

```
git bisect start master 599c577dbd714eedcd3c83a52099adedc3380b98
git bisect run bash -c 'timeout -s INT 2 make serve; [ "$?" == "124" ]'
```

I'm also unable to test changes via `make docker-serve`, meaning we currently have no ability to test out changes to `grpc.io`.

CC @lucperkins 
CC @chalin 
CC @thisisnotapril

--- a/README.md
+++ b/README.md
@@ -2,9 +2,9 @@
 
 This repository houses the assets used to build and deploy the gRPC website, available at https://grpc.io. The site is built using the [Hugo](https://gohugo.io) static site generator. Check out the [Hugo Quick Start](https://gohugo.io/getting-started/quick-start/) for a quick intro.
 
-## Running the site locally
+## Serving the site locally
 
-To run the site locally, you need to [install Hugo](https://gohugo.io/getting-started/installing). Once Hugo is installed:
+To build and locally serve this site, you need to [install Hugo, extended version](https://gohugo.io/getting-started/installing). Once Hugo is installed:
 
 ```bash
 make serve
@@ -18,11 +18,11 @@ make docker-serve
 
 ## Publishing the site
 
-The gRPC website is automatically published by [Netlify](https://netlify.com). Any time changes are pushed to the `master` branch, the site is re-built and re-deployed. This process does not require manual management.
+The gRPC website is _automatically_ published by [Netlify](https://netlify.com). Any time changes are pushed to the `master` branch, the site is rebuilt and redeployed.
 
 ## Site content
 
-All of the [Markdown](https://www.markdownguide.org/) content used to build the site's documentation, blog, etc. is in the [`content`](./content) directory.
+All of the [Markdown](https://www.markdownguide.org) content used to build the site's documentation, blog, etc. is in the [`content`](content) directory.
 
 ## Checking links
 

